docs/words.tex

```diff
@@ -367,6 +378,14 @@ \section{System State}
 : x ;
 forget
 \end{verbatim}
+
+\item[dowords ( xt -- )] Executes \texttt{xt} for successive headers starting from \texttt{latest}. Each time \texttt{xt} is called, it must consume an argument from the stack and leave behind a truth value. If true (not 0), it will continue to the next header address and call \texttt{xt} again. If false, or if it reaches the end of the dictionary, dowords will return. Example:

```

Comment:
I think this text could be clearer. As an example, from this text I am missing information how to implement `xt`, like what arguments it will get passed on the stack. Another example, "return a flag" would be more to the point than "leave behind a truth value".

Maybe [TRAVERSE-WORDLIST](https://forth-standard.org/standard/tools/TRAVERSE-WORDLIST) could be used as a starting point? Just remove the  the parts that are not relevant or too much information.